sql/service/src/main/scala/org/apache/spark/sql/service/cli/operation/SparkGetTypeInfoOperation.scala

```diff
@@ -102,7 +101,7 @@ private[service] class SparkGetTypeInfoOperation(
       parentSession.getUsername)
 
     try {
-      ThriftserverShimUtils.supportedType().foreach(typeInfo => {
+      Type.values().foreach(typeInfo => {
```

Comment:
should we remove INTERVAL_TYPE from Type.java?

Reply:
+1